src/reducers/authSlice.js

```diff
@@ -0,0 +1,70 @@
+import { createSlice, createAsyncThunk } from "@reduxjs/toolkit";
+const axios = require("axios");
+
+export const login = createAsyncThunk(
+  'auth/login', 
+  async (loginData, { rejectWithValue }) => {
+  const response = await axios
+    .post(
+      "http://localhost:8000/graphql", //update
+      {
+        query: `query{ login(
+          email: "${loginData.email}"
+          password: "${loginData.password}"
+        ) {
+        _id
+        name {
+          firstName
+          lastName
+        }
+        token
+        }}`,
+      },
+      {
+        headers: {
+          "Content-Type": "application/json",
+        },
+      }
+    )
+    .catch((error) => {
+      if (error.response) {
+        return error.response.data.errors[0].message;
+      }
+    });
+  if(response.data != undefined) {
+    localStorage.setItem("token", response.data.data.login.token);
```

Comment:
Are we still using `localStorage` to store the auth token?

Reply:
Going with this for now. Will maybe move to Cookies once the basic infrastructure gets in place